install.yaml

```diff
@@ -58,11 +62,8 @@ pre_install_actions:
 # If you use directories, they must be directories that are managed
 # by this add-on, or removal could remove things that are not owned by it
 project_files:
-  - docker-compose.oci8.yaml
```

Comment:
Please add this back in.

Reply:
It was too early in the morning. This is no longer required with your change.

Reply:
@rfay, is it required to add this for the current setup, as it already supports AMD and ARM processors?

Reply:
The entire file is no longer requried. It was just too early in the morning when I was reviewing :)